packages/components/chip/src/Chip.test.tsx

```diff
@@ -1,27 +1,230 @@
-import { render, screen } from '@testing-library/react'
+import { Icon } from '@spark-ui/icon'
+import { InfoFill } from '@spark-ui/icons'
+import { fireEvent, render, screen } from '@testing-library/react'
 import userEvent from '@testing-library/user-event'
 import { describe, expect, it, vi } from 'vitest'
 
 import { Chip } from './Chip'
+import { ChipClearButton } from './ChipClearButton'
+import { ChipContent } from './ChipContent'
+import { ChipLeadingIcon } from './ChipLeadingIcon'
 
 describe('Chip', () => {
   it('should render', () => {
-    render(<Chip>Hello World!</Chip>)
+    // Given
+    const props = { children: 'Hello World!' }
+
+    render(<Chip {...props} />)
+
+    expect(screen.getByText(props.children)).toBeInTheDocument()
+  })
+
+  it('should be visible', () => {
+    // Given
+    const props = { children: 'Hello World!' }
+
+    render(<Chip {...props} />)
+
+    expect(screen.getByText(props.children)).toBeVisible()
+  })
+
+  it('should render all the elements', () => {
+    // Given
+    const props = { onClear: vi.fn(), onClick: vi.fn() }
+    const contentProps = { children: 'Hello World!' }
+    const clearProps = { label: 'label' }
+    const leadingIconProps = {
+      children: (
+        <Icon label="info">
+          <InfoFill />
+        </Icon>
+      ),
+    }
+
+    render(
+      <Chip {...props}>
+        <ChipLeadingIcon {...leadingIconProps} />
+        <ChipContent {...contentProps} />
+        <ChipClearButton {...clearProps} />
+      </Chip>
+    )
+
+    expect(screen.getByText(contentProps.children)).toBeInTheDocument()
```

Comment:
You are only checking for "Hello World!" here like in the previous test, it doesn't test the presence of the Leading/Trailing icons.

Reply:
done